### DISCUSSION TITLE
NPC 캐릭터 위치 지정

### DISCUSSION THREAD

**Discussion by esang28** [naverz/zepeto-studio-kor#584] [5. Q&A]
목적: NPC 캐릭터 로드 시 좌표로 위치 지정

안녕하세요, 코딩 초보입니다.

제페토 스튜디오 가이드에 따라 빈 오브젝트에 아래와 같은 타입스크립트를 컴포넌트 창에 추가해 NPC를 로드하였습니다.
플레이를 하면 플레이어 캐릭터가 생성되는 위치에 NPC가 함께 로드 되는데, Vector3 좌표값을 적어 해당 좌표에 NPC를 위치하려면 코드를 어떻게 수정해야하는지 알려주시면 고맙겠습니다.

import { ZepetoScriptBehaviour } from 'ZEPETO.Script'
import { SpawnInfo, ZepetoPlayers, LocalPlayer, ZepetoCharacter,ZepetoCharacterCreator } from 
'ZEPETO.Character.Controller'
import { GameObject, Object, Vector3 } from 'UnityEngine';

export default class NPCscript extends ZepetoScriptBehaviour {

    Start() {    
        
        ZepetoCharacterCreator.CreateByZepetoId("[ZEPETO_ID]", new SpawnInfo(), (character : ZepetoCharacter)=>{
   // Handling Character Instance..
    })

}

**Comment by x4cf** [comment]
함수에 넣는 스폰인포를 요렇게 만들어서 넣어보시면 좋을것 같아요
var spawnInfo = new SpawnInfo();
spawnInfo.position = new Vector3(0, -10, 0);

**Reply by esang28** [reply]
start () 안에 작성하는게 아닐까요?

    Start() {    

        var spawnInfo = new SpawnInfo();
        spawnInfo.position = new Vector3(-12.5,0.5,25);
        
        ZepetoCharacterCreator.CreateByZepetoId("NPC로사용할아이디", new SpawnInfo(), (character : ZepetoCharacter)=>{ });

         }

play를 하면 움직일 캐릭터와 NPC 캐릭터가 같은 자리에 겹쳐서 생성됩니다...

**Reply by padorami** [reply]
I would like to know the answer to this. Is there a workaround that specifies the location of the npc?
